stanford/cme/edxapp1.yml

```diff
@@ -2,5 +2,6 @@
 ---
 - include: ../playbooks/edxapp.yml
   vars:
-    env: cme
-    machine: app1
+    CLUSTER_INSTANCE_TYPE: 'm4.large'
+    CLUSTER_NUMBER: 1
+    COMMON_DEPLOYMENT: cme
```

Comment:
Just wondering since CLUSTER_NUMBER and COMMON_DEPLOYMENT are defined in cme/group_vars/all, they don't need to be redefined here, right? Or is it just for explicit readability

Reply:
No, it is necessary.

`group_vars` are applied to hosts in a specific group.
The problem is, we use these vars to build out the `hosts` field of playbooks.
vars can't be applied to a host if we haven't yet selected the host :)

Reply:
I see, so this is just the minimum to allow it to then take the rest from group_vars

Reply:
Yeah, exactly. It's kind of a pain and I don't really care for it,
but I couldn't manage to find a cleaner approach :\